### DISCUSSION TITLE
Generic deepgrow -- High CPU usage and low GPU usage

### DISCUSSION THREAD

**Discussion by RafaelPalomar** [Project-MONAI/MONAILabel#282] [Q&A] [answered]
Hello,

I'm using MONAILabel with the 3D Slicer Extension. The training phases are taking very long and I observe high usage of 1 of the CPUs and intermittent use of the GPU. Most of the time is the CPU doing the job. Is this the expected behaviour? I would expect the GPU to be more involved in the training. Thank you in advance.

**Comment by SachidanandAlle** [comment, marked as answer]
the transforms in the example/sample apps are based on CPU.. however you can use/bring your own GPU based transforms to make the best use of available GPUs.  The example/sample apps are only to illustrate the basic workflow through MONAI Label.
